tests/test_imagelogger.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/python3
+# Copyright 2013-2020 Intranet AG and contributors
+#
+# guibot is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# guibot is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with guibot.  If not, see <http://www.gnu.org/licenses/>.
+
+import os
+import unittest
+from PIL.Image import Image
+from unittest.mock import MagicMock, patch
+
+from guibot.imagelogger import ImageLogger
+from guibot.config import GlobalConfig, TemporaryConfig
+
+
+class ImageLoggerTest(unittest.TestCase):
```

Comment:
I think the image logger is tested in `test_finder` where there are checks for the dumped files like (these)[https://github.com/intra2net/guibot/blob/2bdd7e107b463dcae84e1a1fd12bb55681c189be/tests/test_finder.py#L229] but I am not sure how the functionality there could be transferred here without redundancy. I guess the current solution is good enough for now as these are less functional and more unit-al tests of the image logger.

Reply:
So we can keep these then?

Reply:
Yes, it is a small step towards improved modularity so let's take it.